test/probes/tedious.test.js

```diff
@@ -184,17 +195,20 @@ describe('probes.tedious ' + pkg.version, function () {
 
   // Query helper
   function query (fn) {
-    const connection = new Connection({
+    const options = {
       userName: user,
       password: pass,
       server: addr.host,
       port: addr.port,
       options: {
-        database: 'test',
         tdsVersion: '7_1',
         encrypt: false
       }
-    })
+    };
+    if (dbname) {
+      options.options.database = dbname;
```

Comment:
maybe `settings` (instead of 'options') for the const name would make this easier to read ?

Reply:
makes sense.